### PR TITLE
update tutorial pages to match tutorial repo

### PR DESCRIPTION
**Pull request by BryceRussell**
#### What kind of changes does this PR include?

New or updated content

#### Description

Update the tutorial pages to match the [tutorial repo](https://github.com/withastro/blog-tutorial-demo) after upgrading to Astro 3.0

### CODE REVIEW

File: src/content/docs/en/tutorial/5-astro-api/4.mdx [hunk 1]
```diff
@@ -72,24 +72,36 @@ Individuals can subscribe to your feed in a feed reader, and receive a notificat
...
-        site: 'https://my-blog-site.netlify.app',
+        site: context.site,
         items: await pagesGlobToRssItems(import.meta.glob('./**/*.md')),
         customData: `<language>en-us</language>`,
       });
     }
     ```
 
-3. This `rss.xml` document is only created when your site is built, so you won't be able to see this page in your browser during development. Quit the dev server and run the following commands to first, build your site locally and then, view a preview of your build:
+3. Add the `site` property to the Astro config with your site's own unique Netlify URL
```
Comment (ElianCodes) [inline src/content/docs/en/tutorial/5-astro-api/4.mdx:92]:
```suggestion
3. Add the `site` property to the Astro config with your site's own unique Netlify URL.
```

Comment (TheOtterlord) [inline src/content/docs/en/tutorial/5-astro-api/4.mdx:92]: should we remove netlify?
```suggestion
3. Add the `site` property to the Astro config with your site's own unique URL.
```

File: src/content/docs/en/tutorial/5-astro-api/4.mdx [hunk 2]
```diff
@@ -115,13 +127,13 @@ Individuals can subscribe to your feed in a feed reader, and receive a notificat
       </Fragment>
     </PackageManagerTabs>
 
-4. Visit `localhost:3000/rss.xml` and verify that you can see (unformatted) text on the page with an `item` for each of your `.md` files. Each item should contain blog post information such as `title`, `url`, and `description`.
+5. Visit `localhost:3000/rss.xml` and verify that you can see (unformatted) text on the page with an `item` for each of your `.md` files. Each item should contain blog post information such as `title`, `url`, and `description`.
```
Comment (ElianCodes) [inline src/content/docs/en/tutorial/5-astro-api/4.mdx:130]:
```suggestion
5. Visit `localhost:4321/rss.xml` and verify that you can see (unformatted) text on the page with an `item` for each of your `.md` files. Each item should contain blog post information such as `title`, `url`, and `description`.
```

File: src/content/docs/en/tutorial/5-astro-api/4.mdx [hunk 3]
```diff
@@ -72,24 +72,36 @@ Individuals can subscribe to your feed in a feed reader, and receive a notificat
...
+
+    ```js title="astro.config.mjs"
+
+    import { defineConfig } from "astro/config";
+
+    // https://astro.build/config
+    export default defineConfig({
+      site: "https://example.com"
+    });
+    ```
```
Comment (ElianCodes) [inline src/content/docs/en/tutorial/5-astro-api/4.mdx:102]:
``````suggestion
    ```js title="astro.config.mjs" ins={4}
    import { defineConfig } from "astro/config";

    export default defineConfig({
      site: "https://example.com"
    });
    ```
``````

**Review by ElianCodes — COMMENTED** [review]
Thank you so much @BryceRussell! Left a couple of small suggestions for you! after applying LGTM!

**Pull request merged**